### PR TITLE
Configs fix

### PR DESCRIPTION
**Pull request by em-torres**

### CODE REVIEW

File: src/settings/base.py [hunk 1]
```diff
@@ -43,6 +42,7 @@
 	'django.contrib.sessions',
 	'django.contrib.messages',
 	'django.contrib.staticfiles',
+    'rest_framework'
```
Comment (oldani) [inline src/settings/base.py]: Missing indentention here

**Review by oldani — CHANGES_REQUESTED** [review]
Just a minor comment

**Comment by em-torres** [comment]
Added some additional last moment changes. Please re-confirm.

File: src/settings/base.py [hunk 2]
```diff
@@ -30,29 +28,28 @@
...
-	'homepage.apps.HomepageConfig',
-	'login.apps.LoginConfig',
-	'sales.apps.SalesConfig',
-	'django.contrib.admin',
-	'django.contrib.auth',
-	'django.contrib.contenttypes',
-	'django.contrib.sessions',
-	'django.contrib.messages',
-	'django.contrib.staticfiles',
+    'homepage.apps.HomepageConfig',
```
Comment (oldani) [inline src/settings/base.py]: Try to order this or by alphabetical order or (default, third partys and your own apps) order.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
-#PPE DJANGO
-
-Project oriented to create a customers friendly environment to serve mailing and packages necessities from Puerto Plata, Dominican Republic to the world.
+#PPE DJANGO
```
Comment (oldani) [inline README.md]: I think you  could do this better, if you want there is a short course on Udacity for writing READMES.

**Review by oldani — COMMENTED** [review]
Just a few suggestions

File: bootstrap.sh [hunk 4]
```diff
@@ -19,4 +19,4 @@ sudo -u postgres psql -c "ALTER DATABASE ppe_dev OWNER TO ppe"
 
 echo -e "************     Installing Dependencies     *************"
 cd ../../vagrant
```
Comment (johnny243) [inline bootstrap.sh]: What is the absolute path of _vagrant_ dir? /home/vagrant? If so, you could change it to something like:

> cd ~vagrant

**Comment by em-torres** [comment]
Made some additional changes.

**Review by oldani — APPROVED** [review]

**Pull request merged**